contracts/pods/PodFactory.sol

```diff
@@ -0,0 +1,225 @@
+    /// @dev Controller only allows existing admin to change
+    /// @param podId Unique id for the orca pod
+    function getPodAdmin(uint256 podId)
+        external
+        view
+        override
+        returns (address)
+    {
+        return IControllerV1(podController).podAdmin(podId);
+    }
+
+    //////////////////// STATE-CHANGING API ////////////////////
+
+    /// @notice Create a child Orca pod with optimistic timelock. Callable by the DAO and the Tribal Council
+    /// @param _members List of members to be added to the pod
+    /// @param _threshold Number of members that need to approve a transaction on the Gnosis safe
+    /// @param _podLabel Metadata, Human readable label for the pod
+    /// @param _ensString Metadata, ENS name of the pod
+    /// @param _imageUrl Metadata, URL to a image to represent the pod in frontends
+    /// @param _minDelay Delay on the timelock
+    /// @param _podAdmin The admin of the pod - able to add and remove pod members
+    function createChildOptimisticPod(
+        address[] calldata _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string calldata _ensString,
+        string calldata _imageUrl,
+        uint256 _minDelay,
+        address _podAdmin
+    ) external override onlyOwner returns (uint256, address) {
+        uint256 podId = memberToken.getNextAvailablePodId();
+
+        address safeAddress = createPod(
+            _members,
+            _threshold,
+            _podLabel,
+            _ensString,
+            _imageUrl,
+            _podAdmin,
+            podId
+        );
+
+        address timelock = createOptimisticTimelock(
+            safeAddress,
+            _minDelay,
+            podExecutor
+        );
+
+        // Set mapping from podId to timelock for reference
+        getPodTimelock[podId] = timelock;
+        getPodId[timelock] = podId;
+        latestPodId = podId;
+
+        emit CreatePod(podId, safeAddress);
+        return (podId, timelock);
+    }
+
+    /// @notice Create an Orca pod - a Gnosis Safe with a membership wrapper
+    function createPod(
+        address[] calldata _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string calldata _ensString,
+        string calldata _imageUrl,
+        address _podAdmin,
+        uint256 podId
+    ) internal returns (address) {
+        podController.createPod(
+            _members,
+            _threshold,
+            _podAdmin,
+            _podLabel,
+            _ensString,
+            podId,
+            _imageUrl
+        );
+        return podController.podIdToSafe(podId);
+    }
+
+    /// @notice Create an optimistic timelock, linking to an Orca pod
+    /// @param safeAddress Address of the Gnosis Safe
+    /// @param minDelay Delay on the timelock
+    /// @param publicExecutor Non-permissioned smart contract that
+    ///        allows any address to execute a ready transaction
+    function createOptimisticTimelock(
```

Comment:
what about vetoing? The "cancel" functionality should be exposed by some kind of veto controller and veto tribe role imo.

Reply:
Imagining a contract like the PublicExecutor called the VetoController which stores a mapping of which addresses can veto which pods or which roles can veto which pods and exposes the "cancel" functionality through them. I think this is required for the mvp

Reply:
Probably not required for this MVP

Reply:
Love it, was breaking this out into a seperate PR to ease reviewing load